Resources/Prototypes/Recipes/Crafting/Graphs/bots/honkbot.yml

```diff
@@ -0,0 +1,38 @@
+- type: constructionGraph
+  id: HonkBot
+  start: start
+  graph:
+  - node: start
+    edges:
+    - to: bot
+      steps:
+      - prototype: BoxHug
+        icon:
+          sprite: Objects/Storage/boxes.rsi
+          state: box_hug
+        name: box of hugs
+      - prototype: RubberStampClown
+        icon:
+          sprite: Objects/Misc/bureaucracy.rsi
+          state: stamp-clown
+        name: clown's rubber stamp
+        doAfter: 2
+      - prototype: BikeHorn
+        icon:
+            sprite: Objects/Fun/bikehorn.rsi
+            state: icon
+        name: bike horn
+        doAfter: 2
+      - prototype: ProximitySensor
+        icon:
+          sprite: Objects/Misc/proximity_sensor.rsi
+          state: icon
+        name: proximity sensor
```

Comment:
not huge on how many of these require prototypes (prox sensor should at least be a tag) but i suppose its fine

Reply:
Yeah, the clown bot requires clown stuff. I ported it first because it doesn't have custom AI